Jenkinsfile

```diff
@@ -149,7 +149,6 @@ pipeline {
                 allOf {
                     not { branch 'weekly-testing' }
                     expression { env.CHANGE_TARGET != 'weekly-testing' }
```

Comment:
Just to be sure, this shows as a change from last one.  Is this intentional?

Reply:
Yeah, that was the conflict.  It was a result of [this change](https://github.com/daos-stack/daos/pull/1964/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3L149) in the previously landed #1964.